### ISSUE TITLE
login dialog keeps refreshing

### ISSUE THREAD

**Issue by noelo-cohelo** [subclipse/subclipse#81]
After fresh installation of Ubuntu, SVN and Subclipse I'm trying to import a project from SVN repository.
I had some alerts about JavaHL, so I know that probably connectors need to be installed or something like that.

But still, what I would like to report here is when I'm trying to log in (I checked the password), the login window is just disappearing and appearing again - without any feedback message.

**Comment by markphip** [comment]
FWIW, Subclipse does not have much control over this.  The dialog itself is a callback we register with the SVN API and the API decides when to call it.  Likewise, we only provide the information on whether you asked to save the credentials the API provides decides whether or not to save it and how it does it.  I will leave this open but doubting we can do anything with it.

**Comment by markphip** [comment]
No, there generally isn't anything available to display. Try the same thing using the command line client, for example.  When it prompts you for credentials enter invalid ones, all you see is another prompt.  The only information we have available is what the SVN API provides to us.

If you do not have the Console view open, I would also try that.  Anything we have available from the API gets logged there.

**Comment by noelo-cohelo** [comment]
But the credentials are ok. There is something not installed, some connectors I guess.

**Comment by markphip** [comment]
I cannot say whether or not there is anything wrong with your install, but I can say you would not even get as far as a prompt if the API was not somewhat available.  The prompts are shown in response to callbacks from the API, so it has to be successfully loading and working to some degree to get as far as you have.

You ought to be able to go to preferences and Team > SVN to see what adapter is loaded and what version is being reported by the API being used.

Note that in order to use SVN 1.10 you need to be using one of our dev snapshot builds.  The release build requires SVN 1.9 and will fail to load 1.10.  But you would also see an error not a prompt.  I would guess that SVNKit is being used, not JavaHL.

So install the latest snapshot build if you have not as it works with SVN 1.10:

https://github.com/subclipse/subclipse/wiki

**Comment by noelo-cohelo** [comment]
OK, I was wrong - I was giving bad username ;)
But still, if you run SVN from terminal, after 3 attempts it tells you "authentication failed".
I think the Subclipse dialog should behave the same instead of refreshing indefinitely.
